policyengine_canada/variables/gov/provinces/mb/tax/income/credits/mb_basic_personal_amount.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_canada.model_api import *
+
+
+class mb_basic_personal_amount(Variable):
+    value_type = float
+    entity = Person
+    label = "Manitoba basic personal amount"
+    unit = CAD
+    definition_period = YEAR
+    reference = "https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1mb/td1mb-23e.pdf"
+    defined_for = ProvinceCode.MB
+
+    def formula(household, period, parameters):
+        p = parameters(period).gov.provinces.mb.credits
```

Comment:
form says this is only for people employed and pensioners?

Reply:
I believe the basic personal amount has been used as an equivalent to the standard deduction and is applied to all forms of income. While the [law](https://web2.gov.mb.ca/laws/statutes/archive/i010(2020-11-05)e.php?df=2019-06-03) is not very helpful, all calculators found online include self-employment income into the bpa calculation

Reply:
I mean that it doesn't seem like it should be for all people. The formula as-is includes kids, for example.

Reply:
you are right it should be just for the head of the household - as they can claim the spouse amount on the same form as well (bpa of spouse)

Reply:
Do they get double if they have a spouse?

Do they get it if they have no earnings and are not pensioners?

Reply:
They will get a separate amount under the "spouse or common law partner amount" -> separate program

BPA is a non-refundable credit so if they have no earnings then it will equal to 0 (not sure if that answers the question)

Reply:
Non-refundability is separate from the requirement shown on this document; e.g. a non-refundable credit without this requirement could still reduce tax from other income types like dividends.

>Every person employed in Manitoba and every pensioner residing in Manitoba can claim this amount.

Is there more information about eligibility in other sources like the legal code?

Reply:
Neither the income tax act of [Manitoba](https://web2.gov.mb.ca/laws/statutes/archive/i010(2020-11-05)e.php?df=2019-06-03) nor of [other provinces](https://nslegislature.ca/sites/default/files/legc/statutes/income%20tax.pdf) is extremely helpful, 
[Tax calculators](https://turbotax.intuit.ca/tax-resources/ontario-income-tax-calculator.jsp#) seem to exclude the BPA from all kinds of income
This is the reply from gpt: 
"The Basic Personal Amount is a non-refundable tax credit that every Canadian resident is entitled to claim on their income tax return. This credit reduces the amount of tax that an individual pays. The amount is adjusted annually to account for inflation.

Income for the purpose of the Basic Personal Amount generally includes all types of taxable income. This includes employment income, self-employment income, and passive income types like interest, rental, and dividend income. So, yes, dividend income would be included in the calculation for the Basic Personal Amount."

Reply:
I'm still unclear why the form would say "Every person employed in Manitoba and every pensioner residing in Manitoba" if that's not a requirement.

Reply:
That is, I'm looking for a source that states that one can claim it even if they are not employed or receiving a pension.

Reply:
[This](https://hr.mcmaster.ca/app/uploads/2019/06/td1-fill-23e.pdf) source states that on the federal level "Every resident of Canada can enter a basic personal amount". [This](https://www.cbc.ca/news/canada/manitoba/2023-budget-1.6765987) news agency (not a primary source), portrays it as the "the income level at which people start paying tax". For manitoba specifically, the only thing I can find is the [income tax act](https://web2.gov.mb.ca/laws/statutes/archive/i010(2020-11-05)e.php?df=2019-06-03) referencing subsection 3.1 "Individuals liable to pay tax" in the BPA section 4.6(3)